docs/contributing.md

```diff
@@ -0,0 +1,173 @@
+# Contributing and help
```

Comment:
I wonder if we should really keep contributing instructions it in two files. I think one `CONTRIBUTING.rst` would be enough but it's up to you to decide